eventmesh-admin/eventmesh-admin-rocketmq/src/test/java/org/apache/eventmesh/admin/rocketmq/util/UrlMappingPatternTest.java

```diff
@@ -86,8 +88,21 @@ public void testGetParamNames() {
     }
 
     @Test
-    public void testCompile() {
-        //TODO : Fix me to test the method compile(). It is better using Mockito not PowerMockito.
+    public void testCompile() throws NoSuchFieldException, IllegalAccessException {
+        // Mock the compiledUrlMappingPattern field with reflection
+        Pattern mockedPattern = mock(Pattern.class);
+        Field compiledUrlMappingPatternField = urlMappingPattern.getClass().getDeclaredField("compiledUrlMappingPattern");
+        compiledUrlMappingPatternField.setAccessible(true);
+        compiledUrlMappingPatternField.set(urlMappingPattern, mockedPattern);
```

Comment:
It seems unnecessary to `mock(Pattern.class)` for the field `compiledUrlMappingPattern` again. Because `compiledUrlMappingPattern` has already been mocked.
You can use reflection to obtain and directly use the `compiledUrlMappingPattern` field.

Reply:
Thank you! It is really a good idea. I have committed the changes, and please review it at your ease.